packages/jest/src/resultsProcessor.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { addResult, createEmptyTestResult } from '@jest/test-result';
+import { AggregatedResult, AssertionResult, TestResult } from '@jest/test-result/build/types';
+import { errMsgHeader, log } from '@sa11y/common';
+import { A11yError, A11yResult, ConsolidatedResults } from '@sa11y/format';
+
+type FailureDetail = {
+    error?: A11yError;
+};
+
+// Map of test suite name to test results
+const consolidatedErrors = new Map<string, AssertionResult[]>();
+
+/**
+ * Create a test failure result for given a11y failure
+ */
+function convertA11yTestResult(testResult: AssertionResult, a11yResult: A11yResult): AssertionResult {
+    return {
+        ...testResult,
+        // TODO (refactor): extract formatting into its own function.
+        //  - Can this satisfy Formatter interface?
+        //  - Be part of format? (FileFormatter vs ConsoleFormatter)?
+        fullName: `${a11yResult.description}: ${a11yResult.selectors}`,
```

Comment:
It's been a while... this is what the test failure will show up as in GUS right? I forget exactly what is the key used when creating new test failures based on the results JSON file.

Reply:
Yes, Class name and full name form part of the test failure title.